### DISCUSSION TITLE
USB Extension Helped Background Noise / RSSI

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js-ui#3746] [General]
File this in the "no duh, that's what we have been saying the whole time" bucket.

When I moved from a Zooz ZST39 to a UZB7 (largely because I messed up my ZST39) - both plugged straight into my RPi 5 - for whatever reasons my background RSSI got much worse. Around 8+ dBm worse.

Bought one of these to try with my UZB7:
https://www.amazon.com/dp/B00KTDB5AC

Plugged it in, moved the USB drive away from the RPi 5, and my background RSSI improved ~10 dBm across the board (-95 to -105). That's a pretty incredible improvement.

Thanks for the suggestion from those that have said a zillion times to use a USB extension on your controller. Wish I would have listened earlier!

**Comment by robertsLando** [comment]
Thanks for sharing your experience 🙏🏼
